uiowa/uiowa
# Switch from sitenow_find_text to drupal/find_text

Resolves #5282 

Moving from a custom module in our codebase to a shared module on drupal.org (https://www.drupal.org/project/find_text). This PR uninstalls sitenow_find_text, installs find_text, and updates config/permissions.

A followup PR will be needed to remove the sitenow_find_text module from docroot/modules/custom after find_text is in place.





# How to test



- Check that Travis completes a successful build
- `git checkout relocate-find-text && git pull`
- `ddev composer install`
- Sync a site, watch that it completes without error
- Log in, test that Find Text is available in the toolbar, and is functionable
  - double check a search that includes a uiowa headline field, and that it appears in the search results
  - bonus points: masquerade in various roles and check permissions. `webmaster` should have access, and only that role at this time
